src/app/core/admin/hashtags/statistics.component.scss

```diff
@@ -0,0 +1,50 @@
+@use './src/styles/colors';
+@use './src/styles/typography';
+@use './src/styles/breakpoints' as bp;
+
+h1 {
+  @extend .text-h1;
+  text-align: center;
+  margin: 20px auto;
+}
+
+hr {
+  width: 80%;
+  border-color: black;
+  margin: 0 auto;
+}
+
+.statistics {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.spinner {
+  margin: 30px auto;
+}
+
+.table {
+  @extend .text-body;
+  @extend .bg-gradient-neutral-3;
+  width: 100%;
+  border-radius: 10px;
+  margin: 30px;
+}
```

Comment:
Czy jeśli się doda te klasy do `div` w html to nie działają?

Reply:
działają ale inaczej, dodając w ten sposób ma jednolity kolor tak jak w makietach, natomiast przypisując w htmlu kazda kolumna ma inny odcień